ansible/group_vars/all.yml

```diff
@@ -1,12 +1,10 @@
 ---
 
-arch_info: "{{ 's390x' if 's390x' in test_platform else 'x86'}}"
+arch_info: "{{ 's390x' if 's390x' in test_platform else 'ppc64le' if 'ppc64le' in test_platform else 'x86'}}"
 
 gcp_ssh_key_file: "{{ lookup('env', 'GCP_SSH_KEY_FILE', default='~/.ssh/google_compute_engine') }}"
 ibmcloud_ssh_key_file: "{{ lookup('env', 'IBMCLOUD_SSH_KEY_FILE', default='~/.ssh/acs-sshkey_rsa.prv') }}"
-
```

Comment:
I think we should keep these empty lines

Reply:
noted